python_src/src/lamp_py/performance_manager/pipeline.py

```diff
@@ -15,7 +15,7 @@
 from lamp_py.runtime_utils.env_validation import validate_environment
 from lamp_py.runtime_utils.process_logger import ProcessLogger
 
-from lamp_py.tableau.pipeline import start_parquet_updates
+from lamp_py.tableau import start_parquet_updates
```

Comment:
this is the only part i don't like. if we import directly from the file we're doing to get the import error and it felt like catch that error should be left to the subdir.